corehq/apps/es/tests/test_case_search_adapter.py

```diff
@@ -89,3 +101,33 @@ def test_index_not_writes_to_bha_adapter_if_not_required(self):
 
         self.assertEqual(docs_in_bha, [])
         self.assertEqual(len(docs_in_case_search), 1)
+
+    def test_bulk_with_bha_mutliplexing(self):
+        with patch('corehq.apps.es.case_search.multiplex_to_adapter', return_value=case_search_bha_adapter):
+            case_search_adapter.bulk(self.bulk_index_actions, refresh=True)
+
+        # Cleanup
+        self.addCleanup(case_search_adapter.bulk, self.bulk_delete_actions, refresh=True)
+        self.addCleanup(case_search_bha_adapter.bulk, self.bulk_delete_actions, refresh=True)
```

Comment:
Could this live in the setUp method instead of here? I think it just gets in the way of understanding what the test is testing.

Reply:
This isn't a blocking comment and I don't plan to make any changes to the current PR related to this, just wanted to flag it.